test/fixtures/sitemap.xml

```diff
@@ -2,10 +2,10 @@
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
-   <url>
-       <loc>https://hex.pm/packages/postgrex</loc>
-       <lastmod>2014-04-17T14:00:00Z</lastmod>
-       <changefreq>daily</changefreq>
-       <priority>0.8</priority>
-   </url>
+    <url>
```

Comment:
it was mixed, 3 spaces in the first level and 4 in the 2nd level; now it's 4 levels everywhere and same as wikipedia article on sitemaps (at least the 1st code snippets) :-) I'll change it to 3 spaces everywhere since this is what sitemaps.org seems to be using